tasks/main.yml

```
@@ -70,27 +70,27 @@
 
     - name: Build Redis.
       become: yes
-      make:
+      ansible.builtin.make:
```

Dear Tobias @tobiashuste, I think this should be the same as the follow-up make call:

```suggestion
      community.general.make:
```

---

Thanks, I missed that!